app/code/Magento/Catalog/Test/Unit/Model/Product/Option/Validator/DefaultValidatorTest.php

```diff
@@ -84,7 +84,7 @@ public function testIsValidTitle($title, $type, $priceType, $price, $product, $m
         $valueMock->expects($this->once())->method('getTitle')->will($this->returnValue($title));
         $valueMock->expects($this->any())->method('getType')->will($this->returnValue($type));
         $valueMock->expects($this->once())->method('getPriceType')->will($this->returnValue($priceType));
-        $valueMock->expects($this->once())->method('getPrice')->will($this->returnValue($price));
+        $valueMock->expects($this->never())->method('getPrice')->will($this->returnValue($price));
```

Comment:
If the method should never be called, the `will` is not necessary (and will probably confuse the reader). 